.pre-commit-config.yaml

```diff
@@ -20,34 +21,14 @@ repos:
     hooks:
       - id: flake8
         additional_dependencies: [
```

Comment:
Why are you keeping these flake8 hooks?

Reply:
I'm keeping things backwards-compatible. Any changes can be made after merge.